Dockerfile

```diff
@@ -3,6 +3,11 @@ FROM node:18 as node
 WORKDIR /app
 COPY web web
 
+# Copy branding (if exists)
+COPY 'brandin[g]/manifest.json' /app/web/assets/
+COPY 'brandin[g]/favicon.ico' /app/web/assets/
+COPY 'brandin[g]/logo.svg' /app/web/assets/img/
```

Comment:
Would it be possible to mount these with a volume like the branding.yaml?
End users should not be required to build the app themselves imo. 

Changes look great otherwise!

Reply:
Good idea. Will look into it!